features/apt_messages.feature

```diff
@@ -605,7 +605,8 @@ Feature: APT Messages
     Scenario Outline: AWS URLs
         Given a `<release>` machine with ubuntu-advantage-tools installed
         When I run `apt-get update` with sudo
-        When I run `pro refresh messages` with sudo
+        When I run `apt-get install ansible -y` with sudo
```

Comment:
Just double checking, why are we installing ansible here ?

Reply:
For the bionic version of this test, there needs to be a package with an update available in esm-apps to get the message